type-definitions.js

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
 
 export type Component = {
     displayName: string,
     description: string,
     demos: { title: string, render: () => React.Element<*> }[],
-    hideHeader?: boolean,
-    exitButtonStyle?: StyleSheet.Style,
+    renderHeader?: ?() => React.Element<*>,
```

Comment:
Can you please annotate the function arguments here? Seems like `renderHeader` gets passed in the `props`?